src/assets/i18n/de-CH.json

```diff
@@ -64,5 +64,26 @@
 
   "activity.action.create": "hinzugefügt",
   "activity.action.modify": "bearbeitet",
-  "activity.action.delete": "gelöscht"
+  "activity.action.delete": "gelöscht",
+
+  "AAP":"Ausaperung",
```

Comment:
If possible we should not translate the IDs but the phases retrieved from masterdataService to avoid duplicate translations.